src/Apps/ArtQuiz/ArtQuizResultsLoader.tsx

```diff
@@ -0,0 +1,70 @@
+import {
```

Comment:
This component should live in the `Apps/ArtQuiz/Components` folder ([see this faq](https://github.com/artsy/force/blob/main/docs/best_practices.md#keep-file-structure-organized) for more info!)

Reply:
resolved